Production/scripts/ShuffleMergeLaw/analysis/framework.py

```diff
@@ -0,0 +1,90 @@
+# coding: utf-8
```

Comment:
- Move the whole project into Analysis/law/ShuffleMerge
- `framework.py` seems to be independent on ShuffleMerge - put it directly in `Analysis/law`, so it can be reused for other job submissions in the future.
- is it necessary to have an empty `__init__.py`?

Reply:
__init__.py is needed to run the "law index" command, which otherwise fails.

Reply:
I moved the project to Analysis/law 9a67a29beff0bc8c5878b9bff139073287a1c33d